### DISCUSSION TITLE
希望添加动态翻页功能

### DISCUSSION THREAD

**Discussion by mary1937** [the1812/Bilibili-Evolved#1788] [功能建议 / Ideas]
现有瀑布流消耗资源太多了，关注的人比较多，想看两天前的动态简直是受苦，卡顿的厉害，3700x+32G内存也hold不住

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved/discussions/1748#discussioncomment-640239

**Reply by the1812** [reply]
(这个不是评论区的吗, 还是说动态卡片也能用?

**Reply by Geeyun-JY3** [reply]
啊这说的是动态首页？那没事了（
